src/System.Reflection/tests/System.Reflection.Tests.csproj

```diff
@@ -25,6 +27,9 @@
     <Compile Include="TypeInfoTests.cs" />
     <Compile Include="ExceptionTests.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard'">
+    <Compile Include="FieldInfoTests.netstandard.cs" />
```

Comment:
>FieldInfoTests.netstandard.cs [](start = 22, length = 29)

is it good choice to name the file with version less netstandard part? in the future if we adding more APIs and more tests, this name will not reflect that and we may create another file with other names. @weshaggard what you think about naming the files like that?